simapp/sims/simulation_helpers.go

```diff
@@ -49,6 +49,7 @@ func SetupSimulation(config simtypes.Config, dirPrefix, dbName string, verbose,
 	return db, dir, logger, false, nil
 }
 
+// Can move to simapp
```

Comment:
we broke everyone two versions ago (v0.47) to move all of those from simapp to the sdk and made simapp as hard as possible to import (as it is our testing and demo app, so it shouldn't be imported).
I am not sure we should move this back.

Reply:
hmm the only reason I moved it to simapp is we just use `SetupSimulation` for integration & some simapp tests. 
Let me see if I can move to x/simulate. How does it sound?

Reply:
Right, but those helpers should be used by chains